src/NServiceBus.Metrics/ProbeBuilders/ReceivePerformanceDiagnosticsBehavior.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus.Pipeline;
+
+class ReceivePerformanceDiagnosticsBehavior : IBehavior<IIncomingPhysicalMessageContext, IIncomingPhysicalMessageContext>
```

Comment:
Hmm the name of the behavior has [not changed](https://github.com/Particular/NServiceBus.Metrics/blob/develop/src/NServiceBus.Metrics/Performance/ReceivePerformanceDiagnosticsBehavior.cs) with this PR so I assume it's fine.